### PR TITLE
html update

### PR DESCRIPTION
**Pull request by 00USER1X**

### CODE REVIEW

File: lesson_01/Mekhi html/mekhi.html
Comment (anthonydmays) [inline lesson_01/Mekhi html/mekhi.html:1]: There are significant differences between your HTML version and your MARKDOWN version. Continue to iterate on this until they are nearly identical. Good progress though!

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: MekhiReadMe/readMe.md
Comment (HeirJordan3) [inline MekhiReadMe/readMe.md:1]: Still see some significant differences from the ReadMe file to the HTML file.

**Review by HeirJordan3 — CHANGES_REQUESTED** [review]

**Review by 00USER1X — COMMENTED** [review]
fixed differences

**Review by anthonydmays — APPROVED** [review]
Images are still incorrect.

**Review by anthonydmays — APPROVED** [review]

**Pull request merged**